components/auto-complete/tests/auto-complete.spec.tsx

```diff
@@ -65,6 +65,23 @@ describe('AutoComplete', () => {
     expect(getValue(wrapper.find('.vxp-auto-complete__input'))).toEqual('')
   })
 
+  it('filter', async () => {
+    const wrapper = mount(AutoComplete, {
+      props: {
+        options: OPTIONS,
+        filter: true,
+        ignoreCase: true
+      }
+    })
+    const inputEl = wrapper.find('input')
+    const value = OPTIONS[0]
+
+    await inputEl.setValue(value.toLocaleUpperCase())
+    const selectPopperEl = wrapper.find('.vxp-select__popper')
+    expect(selectPopperEl.exists()).toBe(true)
+    expect(selectPopperEl.find('.vxp-select__label').text()).toEqual(value)
```

Comment:
I think here should also test if the filtered options count equals to `1`.

Reply:
ok